local-process-quickstart.sh

```diff
@@ -70,8 +82,17 @@ while getopts "g:n:r:cd" opt; do
    esac
 done
 
+
+if [ -z "$BIKENS" ]; then
+   echo "Defaulting Kubernetes Namespace to: bikeapp"
+   BIKENS="bikeapp"
+fi
+BIKENS=${BIKENS,,}
```

Comment:
What does this do? On my Mac, it gives an error `bad substitution`. What were you trying to do?

Reply:
I was allowing the namespace to be passed as an option - so was _making the namespace lowercase_.  It's only supported in newer bash versions, i didn't check Mac - so lets bin line 90

Reply:
Ah. The cross-platform way that will work for that is `echo $BIKENS | tr [:upper:] [:lower:]`.